CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+## 3.7.0-RELEASE
+* Update to `NotificationsAPIClient.sendSms()`
+    * added `smsSenderId`: an optional smsSenderId specified when adding text message senders under service settings, if this is not provided the default text message sender for the service will be used. `smsSenderId` can be omitted.
```

Comment:
Does one of these need to be the JSON name i.e. added `smsSenderId`: an optional sms_sender_id?

If not the email one needs updating